spec/controllers/proposals_controller_spec.rb

```diff
@@ -229,9 +229,8 @@
       token = create(:api_token, step: approval)
       approval.user.add_delegate(create(:user))
 
-      post :approve, id: proposal.id, cch: token.access_token
+      get :approve, id: proposal.id, cch: token.access_token
```

Comment:
if you read over the other tests in that file, you'll see that `get` will only work with an access_token, while `post` is required otherwise.